src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-clients/cephfs-clients.component.spec.ts

```diff
@@ -4,7 +4,15 @@ import { RouterTestingModule } from '@angular/router/testing';
 
 import { BsDropdownModule } from 'ngx-bootstrap/dropdown';
 
-import { configureTestBed, i18nProviders } from '../../../../testing/unit-test-helper';
+import { By } from '@angular/platform-browser';
+import { ToastModule } from 'ng2-toastr';
```

Comment:
We are now using a different package.
```suggestion
import { ToastrModule } from 'ngx-toastr';
```